src/utils.rs

```diff
@@ -844,3 +869,337 @@ pub fn parse_params_str(params_strs: String) -> Result<Vec<Params>, String> {
 
     Ok(unique_params.into_iter().collect())
 }
+
+// this should be replaced with branchwater's MultiCollection when it's ready
```

Comment:
maybe create an issue in branchwater repo? I didn't really design around re-use 😅 - you could also link to https://github.com/sourmash-bio/sourmash/issues/3321.

so, ok, suggest:
- create issue in this repo
- link to sourmash issue https://github.com/sourmash-bio/sourmash/issues/3321

Reply:
sure, will do. Note, it doesn't really need to be replaced with `MultiCollection` - just, if we eventually get `MultiCollection` in sourmash core, it'd be good to just have one supported implementation.